dm/unit/unit.go

```diff
@@ -65,7 +63,7 @@ type Unit interface {
 // we can refine to add error scope field if needed
 func NewProcessError(err error) *pb.ProcessError {
 	result := &pb.ProcessError{
-		Msg: errors.ErrorStack(err),
+		Msg: "",
 	}
 	if e, ok := err.(*terror.Error); ok {
```

Comment:
When `ok` is false, we should set `result.Msg = err.Error()`

Reply:
Remove errorStack in https://github.com/pingcap/dm/blob/master/dm/master/server.go#L1521

Reply:
@lichunzhu done